ERCS/erc-7815.md

```diff
@@ -0,0 +1,263 @@
+---
+eip: 7815
+title: Swap Order Routing Interface
+description: An interface to simulate over swap liquidity.
```

Comment:
I'd definitely drop "An interface", and maybe elaborate a bit on the content of the proposal. Perhaps:

```suggestion
description: Calculate price, swap tokens, retrieve trading limits, and query capabilities of liquidity pools.
```